modules/integration/src/test/scala/scala/cli/integration/RunScriptTestDefinitions.scala

```diff
@@ -549,4 +551,65 @@ trait RunScriptTestDefinitions { _: RunTestDefinitions =>
       expect(p.out.trim() == "42")
     }
   }
+
+  test("verify os.Path attributes") {
+    // requires os-lib 0.9.2 or later to succeed in Windows
+    val dr      = os.Path.driveRoot
+    val testStr = "/omg"
+    val p       = os.Path(testStr) // <<< must not throw Exception
+    val absPath = p.toString.replace('\\', '/')
+    val relPath = if (dr.nonEmpty) absPath.drop(2) else absPath
+    val synPath = s"${os.Path.driveRoot}$relPath"
+    printf("absPath[%s]\n", absPath)
+    printf("synPath[%s]\n", synPath)
+    printf("relPath[%s]\n", relPath)
+    expect(absPath == synPath)
+    expect(relPath == testStr)
+    expect(absPath endsWith testStr)
+  }
+
+  test("verify drive-relative JAVA_HOME works") {
+    val java8Home =
+      os.Path(os.proc(TestUtil.cs, "java-home", "--jvm", "zulu:8").call().out.trim(), os.pwd)
+
+    val dr = os.Path.driveRoot
+    printf("driveRoot: %s\n", dr)
```

Comment:
```suggestion
```
unnecessary dev logs, no need to commit those

Reply:
side note: there's no need to log here, but why use `printf` in the first place?
It's a lot more readable to just use `println`, no?
for example, you'd be able to just: 
```scala
println(s"driveRoot: $dr")
```